packages/trade/exchange-io-erc1888/src/deposit-watcher/deposit-watcher.service.ts

```diff
@@ -90,7 +90,7 @@ export class DepositWatcherService implements OnModuleInit {
 
         const { transactionHash } = event;
 
-        const { _from: from, _to: to, _value: value, _id: id } = log;
+        const [, from, to, id, value] = log;
```

Comment:
Please use `_` for unused variable, this `,` is really cryptic

Reply:
Deepscan will complain about the unused variable.